.github/workflows/reusable-secrets-sync.yml

```
@@ -0,0 +1,140 @@
+          if summary:
+              with open(summary, "a", encoding="utf-8") as fh:
+                  fh.write(f"## Secrets Sync — `{name}`\n\n")
+                  fh.write(f"- Targets: {len(records)}\n")
+                  synced = sum(1 for r in records if r['status'] == 'synced')
+                  fh.write(f"- Synced: {synced}\n")
+                  fh.write(f"- Dry-run: `{dry_run}`\n")
+                  fh.write(f"- Audit log: `{audit_path}`\n")
+          PY
+
+      - name: Commit audit record
+        if: ${{ secrets.AUDIT_PAT != '' }}
```

actionlint (1.7.12)

[error] context "secrets" is not allowed here. available contexts are "env", "github", "inputs", "job", "matrix", "needs", "runner", "steps", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)